src/components/Container/index.js

```diff
@@ -70,10 +74,10 @@ const Container = ({
 Container.defaultProps = {
   boxStyle: undefined,
   disabled: false,
-  edition: false,
+  edition: undefined,
```

Comment:
I don't really understand why you flipped the booleans to `undefined` ? 🤔

Reply:
good point it was a first debug attempt I set it back to boolean